apps/server/src/modules/deletion/builder/deletion-request-log-response.builder.ts

```diff
@@ -1,4 +1,4 @@
-import { DomainOperation } from '@shared/domain/interface';
+import { DomainDeletionReport } from '@shared/domain/interface';
```

Comment:
Should it be placed in shared? Each that want to use this builder must be import the deletion module and over the modul barrel file you can allow it to use the interface if someone has a need on it.

Reply:
@CeEv hm I think it depends on the architecture we want to have within the repository. @WojciechGrancow placed these interfaces in the `shared` module to limit the dependency on the _Deletion module_. But if you want we can move all these shared interfaces into the _Deletion module_ (but then we'll have direct dependency on the _Deletion module_). Is there any general rule for that already? Do we want to get rid of this `shared` module completely?

Reply:
@Metauriel Ok got it. @WojciechGrancow please move all the shared interfaces in to the _Deletion module_ and export them on module's barrel file.